src/main/java/uk/gov/digital/ho/hocs/workflow/api/FormService.java

```diff
@@ -22,25 +26,41 @@ public class FormService {
 
     private final ScreenRepository screenRepository;
 
-    public FormService(CaseworkClient caseworkClient, ScreenRepository screenRepository) {
+    private final List<String> additionalScreenFolders;
+
+    public FormService(CaseworkClient caseworkClient,
+                       ScreenRepository screenRepository,
+                       @Value("${hocs.screens.additionalFolders}") List<String> fieldFolders) {
         this.caseworkClient = caseworkClient;
         this.screenRepository = screenRepository;
+
+        this.additionalScreenFolders = fieldFolders;
+        this.additionalScreenFolders.add("live"); //TODO: magic word - can/do we extract this?
```

Comment:
Conversation to be had about whether we just have this as a default here - or we embed it directly into the deployment properties.